src/util/RedisBinary.ts

```diff
@@ -40,8 +40,8 @@ export default class RedisBinary {
 
       log(`RedisBinary: found system binary path at "${systemBinary}"`);
       binaryPath = systemBinary;
-    } catch (err) {
-      log(`RedisBinary: can't find system binary at "${systemBinary}".\n${err.message}`);
+    } catch (err: any) {
```

Comment:
can we use `Error` instead of `any`?

Reply:
linter says:

  Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)